docs/admin/kubeadm.md

```diff
@@ -389,6 +389,17 @@ configuration file.
 If `KUBE_KUBERNETES_DIR` is specified, you may need to rewrite the arguments of the kubelet.
 (e.g. --kubeconfig, --pod-manifest-path)
 
+If `KUBE_REPO_PREFIX` is specified, you may need to set --pod-infra-container-image which is the pause image.
```

Comment:
suggest: 

If \`KUBE_REPO_PREFIX\` is specified, you may need to set the kubelet flag \`--pod-infra-container-image\` which specifies which pause image to use.
Defaults to \`gcr.io/google_containers/pause-${ARCH}:3.0\` where \`${ARCH}\` can be one of \`amd64\`, \`arm\`, \`arm64\`, \`ppc64le\` or \`s390x\`